spec.md

```diff
@@ -1931,7 +1931,13 @@ message NodeStageVolumeRequest {
   // that the path is directory and that the process serving the
   // request has `read` and `write` permission to that directory. The
   // CO SHALL be responsible for creating the directory if it does not
-  // exist.
+  // exist.The CO will pass around the staging_target_directory from
```

Comment:
Thanks for this PR! I like the spirit of this: proper use of the staging directory has come up several times in community discussion because, I think, the spec doesn't go far enough to provide usage examples.

That said, I think the wording here needs a bit of cleanup, as well as punctuation.

Reply:
@jdef thanks.. 
>That said, I think the wording here needs a bit of cleanup, as well as punctuation.
>
Can you please point out the change you suggest. I am happy to take it in as it is :+1:

Reply:
How about something like this:
```
  // The CO MUST pass the same staging_target_directory to 
  // NodeStageVolume and NodePublishVolume. 
  // The SP MAY use staging_target_directory in 
  // any way it chooses, including but not limited to:
  // - as a mount point to stage the volume directly
  // - as a parent directory for a newly created mount point
  // - as a parent directory for an attached device
  // - as a parent directory for metadata files
```

Reply:
@julian-hj yeah, better.. reworded as above. @jdef @saad-ali ptal.